test/apicoverage/tools/main.go

```diff
@@ -44,6 +46,23 @@ func getFailedResourceCoverages() *coveragecalculator.CoveragePercentages {
 		ResourceCoverages: percentCoverages}
 }
 
+// Helper function to cleanup any existing Junit XML files.
+// This is done to ensure that we only have one Junit XML file providing the
+// API Coverage summary.
+func cleanupJunitFiles(artifactsDir string) {
```

Comment:
Can we move this change to knative/pkg? It looks like a common function.

Reply:
Fair-enough moved it to knative/pkg in https://github.com/knative/pkg/pull/719